src/Popover.js

```diff
@@ -34,34 +30,66 @@ const Popover = ({
     dataTest,
     elevation,
     maxWidth,
+    observePopperResize,
+    observeReferenceResize,
     placement,
-    onBackdropClick,
```

Comment:
This makes this a breaking change, can we keep it `onBackdropClick` since this is going into `4.x`?

Reply:
great find